### ISSUE TITLE
@glideapps/glide-data-grid-source fails to build with next.js

### ISSUE THREAD

**Issue by sitch** [glideapps/glide-data-grid#391]
When importing the `@glideapps/glide-data-grid-cells@4.1.1-alpha3` package as:

```ts
import { useExtraCells } from "@glideapps/glide-data-grid-cells"
```

We get a working dev environment, but the following error when running `next build`

```bash
info  - Creating an optimized production build  
Failed to compile.

./node_modules/@glideapps/glide-data-grid-cells/dist/js/index.js
export 'StarCell' (reexported as 'StarCellType') was not found in './cells/star-cell.js' (possible exports: default)

Import trace for requested module:
./src/components/FortressDataGrid/FortressDataGrid.tsx
./pages/app/information-sets/[id]/index.tsx

./node_modules/@glideapps/glide-data-grid-cells/dist/js/index.js
export 'SparklineCell' (reexported as 'SparklineCellType') was not found in './cells/sparkline-cell.js' (possible exports: default)

Import trace for requested module:
./src/components/FortressDataGrid/FortressDataGrid.tsx
./pages/app/information-sets/[id]/index.tsx

./node_modules/@glideapps/glide-data-grid-cells/dist/js/index.js
export 'TagsCell' (reexported as 'TagsCellType') was not found in './cells/tags-cell.js' (possible exports: default)

Import trace for requested module:
./src/components/FortressDataGrid/FortressDataGrid.tsx
./pages/app/information-sets/[id]/index.tsx

./node_modules/@glideapps/glide-data-grid-cells/dist/js/index.js
export 'UserProfileCell' (reexported as 'UserProfileCellType') was not found in './cells/user-profile-cell.js' (possible exports: default)

Import trace for requested module:
./src/components/FortressDataGrid/FortressDataGrid.tsx
./pages/app/information-sets/[id]/index.tsx

./node_modules/@glideapps/glide-data-grid-cells/dist/js/index.js
export 'DropdownCell' (reexported as 'DropdownCellType') was not found in './cells/dropdown-cell.js' (possible exports: default)

Import trace for requested module:
./src/components/FortressDataGrid/FortressDataGrid.tsx
./pages/app/information-sets/[id]/index.tsx


> Build failed because of webpack errors
error Command failed with exit code 1.
```

**Comment by sitch** [comment]
Seems like they might just need `export type` instead of `export`.  See: https://github.com/storybookjs/storybook/issues/13504

**Comment by jassmith** [comment]
Fixed in 4.1.1
